docs/tutorial/images.rst

```diff
@@ -45,9 +43,25 @@ For an even lower-level look at data, the raw photon count samples can be access
     photons = scan.red_photons
     plt.plot(photons.timestamps, photons.data)
 
-The images can also be exported in the TIFF format::
+There are also several properties available for convenient access to the scan metadata
```

Comment:
End with a `:`?

I know that this is not part of this PR, but line 40 has always irked me a bit. I'd suggest changing it from:
```The images contain pixel data where each pixel is made up a multiple photon count samples collected by the scanner.```
to:
```The images contain pixel data where each pixel represents summed photon count samples collected by the scanner.```

or even just `summed photon counts`